rust/src/trans/bootstrap.json

```diff
@@ -193,14 +193,9 @@
         "ret": {"c_type": "FfiResult<AnyTransformation *>"}
     },
     "make_count_by": {
-        "description": "Make a Transformation that computes the count of each unique value in data. \nThis assumes that the category set is unknown. \nThis uses a restricted-sensitivity proof that takes advantage of known dataset size. \nUse `make_resize` to establish dataset size. \nThis transformation depends on a measurement that has not yet been implemented.",
+        "description": "Make a Transformation that computes the count of each unique value in data. \nThis assumes that the category set is unknown.",
         "features": ["contrib"],
         "args": [
-            {
-                "name": "size",
-                "c_type": "unsigned int",
-                "description": "Number of records in input data."
-            },
```

Comment:
Indeed. The new Cargo.toml from the crate merge PR did not invoke the build script. Tests passed because the crate merge PR did not change the public interface, so the python API didn't go stale.